lib/sass/script/value/map.rb

```diff
@@ -57,7 +57,7 @@ def to_sass(opts = {})
 
       to_sass = lambda do |value|
         if value.is_a?(Map) || (value.is_a?(List) && value.separator == :comma)
-          "(#{value.to_sass(opts)})"
+          "#{value.to_sass(opts)}"
```

Comment:
That's weird. The output of inspecting a list contains to parentheses - http://sassmeister.com/gist/5051ea9588061673d807

Also, I wonder why the parens aren't being applied from list's own inspect method: https://github.com/sass/sass/blob/2bdc564dea068c4e19064daea25349d93f8d01b8/lib/sass/script/value/list.rb#L73

Lastly, this bugfix pr is to stable, whereas the other is a feature implementation, so it is submitted to master.